packages/opentelemetry-resources/src/platform/browser/service-name.ts

```diff
@@ -0,0 +1,19 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+export function serviceName(): string {
+  return window.location.host;
```

Comment:
web worker doesn't have the window, use the `globalthis` from api

Reply:
API doesn't export that variable. Can I just use the builtin `globalThis` because we are in browser specific code here or is there browser compatibility issues?

Reply:
hmm will this work (writing from head)
```
  return (typeof globalThis === 'object' ? globalThis : window).location.host;
```